README.md

```diff
@@ -1,14 +1,14 @@
 ### Postfix Redis
-This is a patch to the postfix 2.9.3 MTA to add a lookup table for redis database.
+This is a patch to the postfix 3 MTA to add a lookup table for redis database.
 This patch depends on hiredis client library.
 
-Apply the patch to a vanilla postfix 2.9.3 source tree
+Apply the patch to a vanilla postfix 3 source tree
 
 Copy dict_redis.c, dict_redis.h and hiredis.h files to the src/global directory  
 Patch Makefile.in and mail_dict.c in src/global with the provided diffs
 
 ### Generate make files  
-% make makefiles CCARGS="-DHAS_REDIS -I/usr/local/include/hiredis" AUXLIBS="-L/usr/local/lib -lhiredis"  
+% make makefiles CCARGS="-DHAS_REDIS -I/usr/include/hiredis" AUXLIBS="-L/usr/lib -lhiredis"  
 Change the include and library location for hiredis client library if required.  
 
 ### Compile postfix  
```

Comment:
Which distribution did you use? Would'nt the include directory change from distribution to distribution.

Reply:
I used alpine, but there is a package on ubuntu as well.  Will install it and check it installs to /usr or /usr/local